src/pages/LoginPage.tsx

```diff
@@ -20,14 +22,28 @@ export function LoginPage() {
     }
   }, [user, navigate])
 
-  const submitLoginForm = (e: React.FormEvent) => {
+  const submitLoginForm = async (e: React.FormEvent) => {
     e.preventDefault()
     email.onBlur()
     password.onBlur()
 
-    if (validForm) {
-      login(email.value)
+    if (!validForm) return
+
+    try {
+      const response = await axios.post(
+        `${import.meta.env.VITE_API_BASE_URL}/api/login`,
+        {
+          email: email.value,
+          password: password.value,
+        }
+      )
+
+      const { email: userEmail, name, authToken } = response.data.data
+      login({ email: userEmail, name, authToken })
       navigate(from, { replace: true })
+    } catch (error: any) {
+      const message = error.response?.data.data.message
+      toast.error(typeof message === 'string' ? message : '잘못된 요청입니다.')
     }
```

Comment:
axios에서는 any를 쓰지 않고 아래처럼 사용할 수 있습니다. any 사용했던 부분 모두 수정해주시고, 타입스크립트에서는 최대한 any 사용을 지양했으면 좋겠어요.
```suggestion
    } catch (error) {
      if (error instanceof axios.AxiosError) {
        const message = error.response?.data.data.message
        toast.error(
          typeof message === 'string' ? message : '잘못된 요청입니다.'
        )
      }
    }
```